afteraction-share.html

```diff
@@ -215,24 +215,21 @@ <h2>{{ page.custom_fields.share_slide_title }}</h2>
 		<a
 			class="afteraction-donate-amount afteraction-donate-amount-1 button margin-bottom-large"
 			href="{% if page.custom_fields.afteraction_donate_alternate_url %}{{ page.custom_fields.afteraction_donate_alternate_url }}{% else %}{% include_tmpl templateset.custom_fields.afteraction_donate_url %}{% endif %}?amount=3&source=aa_{{ page.id }}"
-			target="_blank"
 		>$3</a>
 		<a
 			class="afteraction-donate-amount afteraction-donate-amount-2 button margin-bottom-large"
 			href="{% if page.custom_fields.afteraction_donate_alternate_url %}{{ page.custom_fields.afteraction_donate_alternate_url }}{% else %}{% include_tmpl templateset.custom_fields.afteraction_donate_url %}{% endif %}?amount=10&source=aa_{{ page.id }}"
-			target="_blank"
 		>$10</a>
 		<a
 			class="afteraction-donate-amount afteraction-donate-amount-3 button margin-bottom-large"
 			href="{% if page.custom_fields.afteraction_donate_alternate_url %}{{ page.custom_fields.afteraction_donate_alternate_url }}{% else %}{% include_tmpl templateset.custom_fields.afteraction_donate_url %}{% endif %}?amount=35&source=aa_{{ page.id }}"
-			target="_blank"
 		>$35</a>
 	</div>
 
 	<div>
 		<a
 			class="afteraction-donate-ask-no button button-alternate"
-			href="{% if page.custom_fields.after_action_next_step == 'scroll-share' %}{% if templateset.custom_fields.getinvolved_url %}{% include_tmpl templateset.custom_fields.getinvolved_url %}{% else %}https://350.org/get-involved{% endif %}{% else%}#{% endif %}"
+			href="{% if page.custom_fields.after_action_next_step == 'scroll-share' %}{% if page.custom_fields.getinvolved_url %}{{ page.custom_fields.getinvolved_url }}{% else %}https://350.org/science{% endif %}{% else%}#{% endif %}"
```

Comment:
@sukhada just confirming this fallback should be set to /science - is that right?

Reply:
yep - Julia requested here: https://github.com/350org/action-tools/issues/60#issuecomment-1398366721